boards.go

```diff
@@ -29,6 +29,24 @@ type Board struct {
 	ModifiedAt        *string            `json:"modified_at,omitempty"`
 }
 
+// BoardLite represents a simplify dashboard (without widgets, notify list, ...)
+// It's used when we load all boards.
+type BoardLite struct {
+	Title        *string `json:"title"`
```

Comment:
Most often, we prefer putting `omitempty` everywhere to stay consistent, I'd suggest you do that.

Reply:
amended